### PR TITLE
fix(RELEASE-1100): linting issues in publish-index-image-task

### PR DESCRIPTION
**Pull request by happybhati**

### CODE REVIEW

File: internal-services/catalog/publish-index-image-task.yaml
Comment (mmalina) [inline internal-services/catalog/publish-index-image-task.yaml:1]: Can you bump the task version?

**Review by johnbieren — APPROVED** [review]
lgtm once task version is bumped

File: internal-services/catalog/publish-index-image-task.yaml
Comment (happybhati) [reply internal-services/catalog/publish-index-image-task.yaml:1]: Done, thanks!

**Review by johnbieren — APPROVED** [review]

**Comment by happybhati** [comment]
@mmalina , @johnbieren  does something changed? 
`Linters / Run gitlint checks` are failing on both PR
```
Run python -m pip install gitlint
  python -m pip install gitlint
  shell: /usr/bin/bash -e {0}
  env:
    TKN_CLI_VERSION: 0.30.1
error: externally-managed-environment

× This environment is externally managed
╰─> To install Python packages system-wide, try apt install
    python3-xyz, where xyz is the package you are trying to
    install.
    
    If you wish to install a non-Debian-packaged Python package,
    create a virtual environment using python3 -m venv path/to/venv.
    Then use path/to/venv/bin/python and path/to/venv/bin/pip. Make
    sure you have python3-full installed.
    
    If you wish to install a non-Debian packaged Python application,
    it may be easiest to use pipx install xyz, which will manage a
    virtual environment for you. Make sure you have pipx installed.
    
    See /usr/share/doc/python3.1[2](https://github.com/hacbs-release/app-interface-deployments/actions/runs/11347518099/job/31559252864?pr=196#step:3:2)/README.venv for more information.

note: If you believe this is a mistake, please contact your Python installation or OS distribution provider. You can override this, at the risk of breaking your Python installation or OS, by passing --break-system-packages.
hint: See PEP [6](https://github.com/hacbs-release/app-interface-deployments/actions/runs/11347518099/job/31559252864?pr=196#step:3:7)68 for the detailed specification.
Error: Process completed with exit code 1.
```

Do you have any suggestions on how this can be fixed?
Similar discussion [Stackoverflow](https://stackoverflow.com/questions/75608323/how-do-i-solve-error-externally-managed-environment-every-time-i-use-pip-3)
suggest use of virtual environment

**Comment by johnbieren** [comment]
> @mmalina , @johnbieren does something changed? `Linters / Run gitlint checks` are failing on both PR
> 
> ```
> Run python -m pip install gitlint
>   python -m pip install gitlint
>   shell: /usr/bin/bash -e {0}
>   env:
>     TKN_CLI_VERSION: 0.30.1
> error: externally-managed-environment
> 
> × This environment is externally managed
> ╰─> To install Python packages system-wide, try apt install
>     python3-xyz, where xyz is the package you are trying to
>     install.
>     
>     If you wish to install a non-Debian-packaged Python package,
>     create a virtual environment using python3 -m venv path/to/venv.
>     Then use path/to/venv/bin/python and path/to/venv/bin/pip. Make
>     sure you have python3-full installed.
>     
>     If you wish to install a non-Debian packaged Python application,
>     it may be easiest to use pipx install xyz, which will manage a
>     virtual environment for you. Make sure you have pipx installed.
>     
>     See /usr/share/doc/python3.1[2](https://github.com/hacbs-release/app-interface-deployments/actions/runs/11347518099/job/31559252864?pr=196#step:3:2)/README.venv for more information.
> 
> note: If you believe this is a mistake, please contact your Python installation or OS distribution provider. You can override this, at the risk of breaking your Python installation or OS, by passing --break-system-packages.
> hint: See PEP [6](https://github.com/hacbs-release/app-interface-deployments/actions/runs/11347518099/job/31559252864?pr=196#step:3:7)68 for the detailed specification.
> Error: Process completed with exit code 1.
> ```
> 
> Do you have any suggestions on how this can be fixed? Similar discussion [Stackoverflow](https://stackoverflow.com/questions/75608323/how-do-i-solve-error-externally-managed-environment-every-time-i-use-pip-3) suggest use of virtual environment

I've never seen that before, and I am not sure why you are now getting that error

**Comment by happybhati** [comment]
> I've never seen that before, and I am not sure why you are now getting that error

ACK, thanks for looking, Johnny. It was working fine until  Friday, but I'm not sure what changed.

Let's see what @mmalina thinks.

I have created a [draft ](https://github.com/hacbs-release/app-interface-deployments/pull/197)for the fix, but I'm not sure if this is the best way to proceed.

**Pull request merged**